modules/planning/common/frame.h

```diff
@@ -121,11 +121,11 @@ class Frame {
       const double planning_start_time,
       prediction::PredictionObstacles *prediction_obstacles);
 
-  void set_last_planned_trajectory(ADCTrajectory last_planned_trajectory) {
+  void set_current_frame_planned_trajectory(ADCTrajectory last_planned_trajectory) {
     last_planned_trajectory_ = std::move(last_planned_trajectory);
   }
 
-  ADCTrajectory last_planned_trajectory() const {
+  ADCTrajectory current_frame_planned_trajectory() const {
```

Comment:
Return "const ADCTrajectory&". Let the user to decide if they want a copy or a ready only reference.

Reply:
Yeah, will fix it.